appshell/cefclient.cpp

```diff
@@ -95,7 +98,19 @@ void AppGetSettings(CefSettings& settings, CefRefPtr<CefCommandLine> command_lin
       command_line->GetSwitchValue(client::switches::kJavascriptFlags);
     
   // Enable dev tools
-  settings.remote_debugging_port = REMOTE_DEBUGGING_PORT;
+  CefString debugger_port = command_line->GetSwitchValue("remote-debugging-port");
+  if (!debugger_port.empty()) {
+    int port = atoi(debugger_port.ToString().c_str());
+    static const int max_port_num = static_cast<int>(std::numeric_limits<uint16_t>::max());
```

Comment:
What happens if the port is already in use?

Reply:
I checked with `python -m SimpleHTTPServer 9999` and Brackets snatches the port from Python HTTP Server.